pkg/token/README.MD

```diff
@@ -0,0 +1,38 @@
+Instruction for recording response using [GO-VCR](https://github.com/dnaeon/go-vcr) for unit test by jonachang
```

Comment:
probably should remove jonathan's alias :)

Reply:
Fixed